thirdeye/thirdeye-frontend/README.md

```diff
@@ -0,0 +1,52 @@
+# thirdeye-frontend
+
+This README outlines the details of collaborating on this Ember application.
+A short introduction of this app could easily go here.
+
+## Prerequisites
+
+You will need the following things properly installed on your computer.
+
+* [Git](https://git-scm.com/)
+* [Node.js](https://nodejs.org/) (with NPM)
+* [Bower](https://bower.io/)
+* [Ember CLI](https://ember-cli.com/)
+* [PhantomJS](http://phantomjs.org/)
+
+## Installation
+
+* `git clone <repository-url>` this repository
+* `cd thirdeye-frontend`
+* `npm install`
+* `bower install`
+
+## Running / Development
+
+* `ember serve`
+* Visit your app at [http://localhost:4200](http://localhost:4200).
+
+### Code Generators
+
+Make use of the many generators for code, try `ember help generate` for more details
+
+### Running Tests
+
+* `ember test`
+* `ember test --server`
+
+### Building
+
+* `ember build` (development)
+* `ember build --environment production` (production)
+
+### Deploying
```

Comment:
line 23 lists the instructions: 
```
  +## Running / Development
  +* `ember serve`
  +* Visit your app at [http://localhost:4200](http://localhost:4200)
```

I'll also update our internal confluence page